controllers/boxes.py

```diff
@@ -2,6 +2,7 @@
 import logging
 import werkzeug.exceptions
 from odoo.addons.http_routing.models.ir_http import slug
+import wdb
```

Comment:
wdb might be a problem. It should not be in any production environment. @jamescrowley